docs/reference/commandline/run.md

```diff
@@ -61,6 +61,7 @@ parent = "smn_cli"
                                     'host': use the host network stack inside the container
                                     'NETWORK': connects the container to user-created network using `docker network create` command
       --oom-kill-disable=false      Whether to disable OOM Killer for the container or not
+      --oom-score-adj=0             Tune OOM preferences for container (-1000 to 1000)
```

Comment:
does this imply the OOM is owned/inside the container? where instead it's in the kernel (host)